docs/src/dev-guide/building-package.md

```diff
@@ -12,10 +12,17 @@ prebuilt version instead, check out the [releases](https://github.com/y-scope/cl
     extra configuration.
 * Python 3.8 or newer
 * python3-venv
+* python3-dev
 * [Task](https://taskfile.dev/)
 
 ## Setup
 
+Clone the repo and sub-components
+
+```shell
+git clone --recurse-submodules https://github.com/y-scope/clp.git
+```
+
```

Comment:
I think this isn't necessary as long as we run the next line (`tools/scripts/deps-download/init.sh`)?